client/ayon_nuke/plugins/load/load_ociolook.py

```diff
@@ -174,11 +174,13 @@ def _create_group_node(
                 # file path from lut representation
                 extension = ocio_item["ext"]
                 item_name = ocio_item["name"]
+                lut_suffix = ocio_item.get("lut_suffix", "")
 
                 item_lut_file = next(
                     (
                         file for file in all_files
                         if file.endswith(extension)
+                        and lut_suffix in os.path.basename(file)
```

Comment:
Shouldn't that be explicit suffix check?

I guess we do modify the file so the suffix might not be at the end of filename, or?
```suggestion
                        and os.path.basename(file).endswith(lut_suffix)
```

Reply:
Ok so I have forgotten that the suffix sill be added to the lut file during integration since the `outputName` is added to the representation. In this case this is actually working . So yes this is working and your suggestion should too @iLLiCiTiT .